src/styles/variables.ts

```diff
@@ -98,6 +98,7 @@ export default {
     iosSafeAreaInsetsPercentage: 0.7,
     androidSafeAreaInsetsPercentage: 1,
     sideBarWidth: 375,
+    tooltipWidth: 334,
```

Comment:
@davidgelhar, I believe there is a better approach to fixing this. This is an arbitrary number (why 334?), could we add a margin to text container instead?